### PR TITLE
Added flipper component

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
[Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/16302) 

This PR is to add the things needed to `vets-website` for our flipper component for payment history. It [has a corresponding PR](https://github.com/department-of-veterans-affairs/vets-api/pull/5361) in `vets-api` to add the feature toggle.

## Testing done
Tested on local

## Screenshots

> With flipper enabled

Screen Shot 2020-11-19 at 11 59 23 AM

> With flipper disabled

Screen Shot 2020-11-19 at 1 59 53 PM

## Acceptance criteria
- [x] Feature name has been added to vets-api in snake case in config/features.yml
- [x] Feature toggle name has been added to vets-website featureFlagNames.js
- [x] A PR has been requested for merging for each of the items above

### CODE REVIEW

File: src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  } else if (props.includedInFlipper === false) {
+    alertContent = (
+      <>
+        <p>
+          To use this feature, you'll need a Premium <strong>DS Login</strong>{' '}
+          account. Your My HealtheVet or ID.me credentials won't work on the
+          eBenefits website. Go to eBenefits to sign in, register, or upgrade
+          your <strong>DS Login</strong> account to Premium.
+        </p>
+        <a
```
Comment (micahchiang) [inline src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js:26]: You can use the `EbenefitsLink` Component from platform similar to [our 686 cta](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/static-pages/view-modify-dependent/686-cta/containers/Form686CTA.jsx#L17) It has built in analytics firing.

**Review by micahchiang — COMMENTED** [review]
Left a couple comments. I think you need an approval from public websites for this to be merged.

File: src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  let alertType;
+  let headline;
+  let alertContent;
+  if (props.includedInFlipper === undefined) {
+    return <LoadingIndicator message="Loading..." />;
+  } else if (props.includedInFlipper === false) {
+    alertContent = (
+      <>
+        <p>
+          To use this feature, you'll need a Premium <strong>DS Login</strong>{' '}
```
[the remark this answers is not in the document]
Comment (cohnjesse) [reply src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js:21]: updated, the mockup was wrong here.

File: src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+          account. Your My HealtheVet or ID.me credentials won't work on the
+          eBenefits website. Go to eBenefits to sign in, register, or upgrade
+          your <strong>DS Login</strong> account to Premium.
```
[the remark this answers is not in the document]
Comment (cohnjesse) [reply src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js:24]: Same as above

File: src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js (hunk 1 above)
Comment (cohnjesse) [reply src/applications/static-pages/view-payment-history/ViewPaymentHistoryCTA.js:26]: Added

**Review by ncksllvn — APPROVED** [review]

**Review by micahchiang — APPROVED** [review]

**Review by U-DON — APPROVED** [review]

**Pull request merged**